docs/lessons/dance-party.md

```diff
@@ -0,0 +1,511 @@
+            . . . . . . . . . . . . . . . .
+            . . . . . . . . . . . . . . . .
+        `, SpriteKind.Projectile)
+        left.setVelocity(0, Speed)
+        left.setPosition(30, 8)
+    } else if (Lane == 2) {
+        up = sprites.create(img`
+            . . . . . . . . . . . . . . . .
+            . . . . . . . . . . . . . . . .
+            . . 6 6 6 6 6 6 6 6 6 6 6 6 . .
+            . . 6 6 6 6 6 6 1 6 6 6 6 6 . .
+            . . 6 6 6 6 6 1 6 1 6 6 6 6 . .
+            . . 6 6 6 6 1 6 6 6 1 6 6 6 . .
+            . . 6 6 6 1 6 6 6 6 6 1 6 6 . .
+            . . 6 6 6 1 1 1 6 1 1 1 6 6 . .
+            . . 6 6 6 6 6 1 6 1 6 6 6 6 . .
+            . . 6 6 6 6 6 1 6 1 6 6 6 6 . .
+            . . 6 6 6 6 6 1 6 1 6 6 6 6 . .
+            . . 6 6 6 6 6 1 1 1 6 6 6 6 . .
+            . . 6 6 6 6 6 6 6 6 6 6 6 6 . .
+            . . 6 6 6 6 6 6 6 6 6 6 6 6 . .
+            . . . . . . . . . . . . . . . .
+            . . . . . . . . . . . . . . . .
+        `, SpriteKind.Projectile)
+        up.setVelocity(0, Speed)
+        up.setPosition(60, 8)
+    } else if (Lane == 3) {
+        down = sprites.create(img`
+            . . . . . . . . . . . . . . . .
+            . . . . . . . . . . . . . . . .
+            . . 6 6 6 6 6 6 6 6 6 6 6 6 . .
+            . . 6 6 6 6 6 6 6 6 6 6 6 6 . .
+            . . 6 6 6 6 6 1 1 1 6 6 6 6 . .
+            . . 6 6 6 6 6 1 6 1 6 6 6 6 . .
+            . . 6 6 6 6 6 1 6 1 6 6 6 6 . .
+            . . 6 6 6 6 6 1 6 1 6 6 6 6 . .
+            . . 6 6 6 1 1 1 6 1 1 1 6 6 . .
+            . . 6 6 6 1 6 6 6 6 6 1 6 6 . .
+            . . 6 6 6 6 1 6 6 6 1 6 6 6 . .
+            . . 6 6 6 6 6 1 6 1 6 6 6 6 . .
+            . . 6 6 6 6 6 6 1 6 6 6 6 6 . .
+            . . 6 6 6 6 6 6 6 6 6 6 6 6 . .
+            . . . . . . . . . . . . . . . .
+            . . . . . . . . . . . . . . . .
+        `, SpriteKind.Projectile)
+        down.setVelocity(0, Speed)
+        down.setPosition(100, 8)
+    } else {
+        right = sprites.create(img`
+            . . . . . . . . . . . . . . . .
+            . . . . . . . . . . . . . . . .
+            . . 6 6 6 6 6 6 6 6 6 6 6 6 . .
+            . . 6 6 6 6 6 6 6 6 6 6 6 6 . .
+            . . 6 6 6 6 6 6 6 6 6 6 6 6 . .
+            . . 6 6 6 6 6 6 1 1 6 6 6 6 . .
+            . . 6 6 6 6 6 6 1 6 1 6 6 6 . .
+            . . 6 6 1 1 1 1 1 6 6 1 6 6 . .
+            . . 6 6 1 6 6 6 6 6 6 6 1 6 . .
+            . . 6 6 1 1 1 1 1 6 6 1 6 6 . .
+            . . 6 6 6 6 6 6 1 6 1 6 6 6 . .
+            . . 6 6 6 6 6 6 1 1 6 6 6 6 . .
+            . . 6 6 6 6 6 6 6 6 6 6 6 6 . .
+            . . 6 6 6 6 6 6 6 6 6 6 6 6 . .
+            . . . . . . . . . . . . . . . .
+            . . . . . . . . . . . . . . . .
+        `, SpriteKind.Projectile)
+        right.setVelocity(0, Speed)
+        right.setPosition(130, 8)
+    }
+})
+game.onUpdateInterval(2000, function () {
+    Speed += 1
+})
+
+```
+
+
+![Working Game](/static/lessons/dance-party/workinggame.gif)
```

Comment:
for all the images, can you change them to have words separated by dashes? ``working-game`` here. A bit easier to parse later on